llvm/lib/CodeGen/GlobalISel/GISelValueTracking.cpp

```diff
@@ -715,6 +715,9 @@ static bool outputDenormalIsIEEEOrPosZero(const MachineFunction &MF, LLT Ty) {
 void GISelValueTracking::computeKnownFPClass(Register R, KnownFPClass &Known,
                                              FPClassTest InterestedClasses,
                                              unsigned Depth) {
+  if (!R.isVirtual())
+    return;
```

Comment:
This shouldn't be reached in the first place, this is only relevant for COPY

Reply:
fixed